src/rt/sched/cown.h

```diff
@@ -923,7 +919,7 @@ namespace verona::rt
     /// Update priority based on the occurrence of a token message, or replace
     /// the token if it is not in the queue. Return true if the current message
     /// is a token.
-    inline bool check_message_token(Alloc* alloc, MessageBody* curr)
+    inline bool check_token_message(Alloc* alloc, MessageBody* curr)
```

Comment:
If a message without a body, the token message?

Reply:
Yes, it's a stub message